packages/client/src/app/components/site-sidenav/site-sidenav.component.ts

```diff
@@ -84,13 +91,14 @@ export class SiteSidenavComponent implements OnInit {
       password: this.loginFields.password.value, 
       rememberMe: this.loginFields.rememberMe.value
     };
-    this.authService.login(credentials).pipe(first()).subscribe(() => {
+
+    this.store.dispatch([new Auth.Login(credentials), new Navigate(['/home'])]).subscribe(() => {
       this.loadingLogin = false;
-      this.router.navigate(['/home']);
+      location.reload();
```

Comment:
Shouldn't the `Navigate(['/home'])` take care of this? Or am I misunderstanding the `dispatch()` syntax?

Reply:
see conversation in discord